tests/WE2E/WE2E_summary.py

```diff
@@ -68,9 +74,13 @@ def setup_logging(debug: bool = False) -> None:
     else:
         raise ValueError(f'Bad arguments; run {__file__} -h for more information')
 
-    # Calculate core hours and update yaml
-    expts_dict = calculate_core_hours(expts_dict)
-    write_monitor_file(yaml_file,expts_dict)
+    if expts_dict:
```

Comment:
I know this was already here with this name, but somehow putting it in an if-statement like this, I now read "expects dict". Not sure if it matters much.